frontend/viewer/src/lib/services/entry-loader-service.svelte.ts

```
@@ -0,0 +1,334 @@
+      this.#idToIndex.set(entries[i].id, start + i);
+    }
+    this.#loaded.add(batch);
+  }
+
+  storeRange(batches: number[], entries: IEntry[]): void {
+    const startIndex = Math.min(...batches) * this.batchSize;
+    for (let i = 0; i < entries.length; i++) {
+      this.#entries.set(startIndex + i, entries[i]);
+      this.#idToIndex.set(entries[i].id, startIndex + i);
+    }
+    batches.forEach(b => this.#loaded.add(b));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)